templates/shortcode-donor-wall.php

```diff
@@ -57,6 +57,12 @@
 					</h3>
 				<?php endif; ?>
 
+				<?php if ( true === $atts['show_company_name'] && isset( $donation['_give_donation_company'] ) ) : ?>
```

Comment:
Pointing out a possible refactor: View classes are a good way to encapsulate conditional logic. So instead of using `true === $atts[ 'foo' ]` this can be wrapped in a class method, like `shouldShowCompay()`, which can then also be re-used within the view template.

Again, @alaca good job matching the current implementation. I like refactoring in a separate step.

Reply:
@kjohnson I agree, but for this particular case, I followed the style of the previous dev.

Reply:
Of course - that was the right decision. Good job.